CHARTER.md

```diff
@@ -1,51 +1,125 @@
+members have voted, the pull request is merged into the `main` branch (always, whether
+approved or rejected). The chairs then record the decision, dissenting comments, and
+other relevant information in the [`RECORD.md`](art-decisions/RECORD.md) file via
+another pull request that is reviewed and approved by at least 50% of the chairs to
+ensure accuracy.
 
-If the majority cannot be reached, the proposal is considered rejected and can be brought to the ART again 2 months after the rejection at the earliest. If the same decision has been rejected twice, it cannot be brought to the ART again.
+## Chair Elections
 
-Any voting period must last at least 7 days to account for different time zones and other obligations, and should last no longer than 30 days. The voting period should be stated in the proposal.
+If a chair leaves through non-participation, through their own resignation, or due to an
+incapacity to carry out their duties, elections for a new chair take place.
 
-ART members cast their votes by starting a review that either approves or rejects the decision suggested via a pull request. Once the voting period is over or all ART members have voted, the chairs record the decision, objections, and other relevant information in the `art-decisions` folder. Silence equals abstaining from a vote.
+After the announcement of a chair leaving, there is a period of 30 days for nominations
+(from the date of the announcement). Any ART member can nominate another ART member or
+themselves, while a second member has to sponsor the nomination. Only members that have
+been part of the ART for at least six months are eligible to become a chair or sponsor a
+nomination.
 
-## Chair Elections
+After the nomination period, elections take place asynchronously through the same
+process as standard proposals.
 
-If a chair leaves through non-participation or through their own announcement, elections for a new chair take place.
+A proposal to remove a chair from their position for any reason other than those stated
+at the beginning of this section follows the normal proposal process.
 
-After the announcement of a chair leaving, there is a period of 30 days for nominations (from the date of the announcement). Any ART member can nominate another ART member or themselves, while a second member has to sponsor the nomination. Only members that have been part of the ART for 6+ months are eligible to become a chair or sponsor a nomination.
+## Code of Conduct Enforcement
 
-After the nomination period, elections take place asynchronously through the same process as standard decision proposals.
+Removing or disciplining an ART member due to a violation of the [Arcalot Code of
+Conduct](https://github.com/arcalot/.github/blob/main/CODE_OF_CONDUCT.md) is initiated
+by any chair as a pull request to the ART. Only the chairs vote on these matters, with
+an approval by a majority of the chairs voting in favor with none voting against. In the
```

Comment:
I think it would be good to tighten this up:  one reading of the phrase "a majority of the chairs voting in favor with none voting against" suggests that the measure passes as soon as more than half of the chairs have voted, so long as none have voted against; however, this doesn't account properly for chairs who have not yet voted (who might, for instance, vote against).  The time limit (which I agree with removing) addressed this confusion, so, without it, I think we need to state the quorum requirement...and I would recommend that **all** chairs be required to "respond" (i.e., with a yeah, nay, or an explicit notice of abstention).

Yes, this creates an opening for a chair to deliberately hang the process by failing to respond, but, in that unlikely occurrence, I think we have a process for removing that person from that role, which would then allow this process to conclude.